extensions/sql-migration/src/dialog/skuRecommendationResults/skuRecommendationResultsDialog.ts

```diff
@@ -0,0 +1,513 @@
+			}
+		];
+
+		const createRow = (dimension: string, value: string) => {
+			const row: azdata.DeclarativeTableCellValue[] = [
+				{ value: dimension },
+				{ value: value }
+			];
+			return row;
+		};
+		const cpuRow = createRow(constants.CPU_REQUIREMENT, constants.CPU_CORES(instanceRequirements?.cpuRequirementInCores!));
+		const memoryRow = createRow(constants.MEMORY_REQUIREMENT, constants.GB(instanceRequirements?.memoryRequirementInMB! / 1024));
+		const dataStorageRow = createRow(constants.DATA_STORAGE_REQUIREMENT, constants.GB(instanceRequirements?.dataStorageRequirementInMB! / 1024));
+		const logStorageRow = createRow(constants.LOG_STORAGE_REQUIREMENT, constants.GB(instanceRequirements?.logStorageRequirementInMB! / 1024));
+		const dataIOPSRow = createRow(constants.DATA_IOPS_REQUIREMENT, constants.IOPS(instanceRequirements?.dataIOPSRequirement!));
+		const logsIOPSRow = createRow(constants.LOGS_IOPS_REQUIREMENT, constants.IOPS(instanceRequirements?.logIOPSRequirement!));
+		const ioLatencyRow = createRow(constants.IO_LATENCY_REQUIREMENT, instanceRequirements?.ioThroughputRequirementInMBps! < 5 ? constants.NA : constants.MS(instanceRequirements?.ioLatencyRequirementInMs!));
+		const storagePropertiesTableRows: azdata.DeclarativeTableCellValue[][] = [
+			cpuRow,
+			memoryRow,
+			dataStorageRow,
+			logStorageRow,
+			dataIOPSRow,
+			logsIOPSRow,
+			ioLatencyRow,
+		];
+
+		const storagePropertiesTable: azdata.DeclarativeTableComponent = _view.modelBuilder.declarativeTable().withProps({
+			ariaLabel: constants.RECOMMENDED_TARGET_STORAGE_CONFIGURATION,
+			columns: columns,
+			dataValues: storagePropertiesTableRows,
+			width: 300
+		}).component();
+
+		const container = _view.modelBuilder.flexContainer().withLayout({
+			flexFlow: 'column'
+		}).withItems([
+			storagePropertiesSection,
+			storagePropertiesTable,
+		]).component();
+		return container;
+	}
+
+	public async openDialog(dialogName?: string, recommendations?: mssql.SkuRecommendationResult) {
+		if (!this._isOpen) {
+			this.instanceRequirements = recommendations?.instanceRequirements;
+
+			switch (this._targetType) {
+				case MigrationTargetType.SQLMI:
+					this.targetRecommendations = recommendations?.sqlMiRecommendationResults;
+					break;
+
+				case MigrationTargetType.SQLVM:
+					this.targetRecommendations = recommendations?.sqlVmRecommendationResults;
+					break;
+
+				case MigrationTargetType.SQLDB:
+					this.targetRecommendations = recommendations?.sqlDbRecommendationResults;
+					break;
+			}
+
+			this._isOpen = true;
+			this.dialog = azdata.window.createModelViewDialog(this.title!, 'SkuRecommendationResultsDialog', 'medium');
+
+			this.dialog.okButton.label = SkuRecommendationResultsDialog.OpenButtonText;
+			this._disposables.push(this.dialog.okButton.onClick(async () => await this.execute()));
+
+			this.dialog.cancelButton.hidden = true;
+			// this.dialog.cancelButton.label = SkuRecommendationResultsDialog.CreateTargetButtonText;
```

Comment:
can this be removed

Reply:
Yes, but the fact that we have to use the cancel button to serve as our actual action button because of gaps in ADS isn't super intuitive, so I'd prefer to leave the comment in so that we remember when we get back to this in the future